### PR TITLE
Fix: Annotations upgrade script

### PR DESCRIPTION
**Pull request by pramodsum**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @pramodsum has signed the CLA. Thanks for the pull request!

File: build/upgrade_annotations.sh [hunk 1]
```diff
@@ -50,6 +50,12 @@ reset_to_master() {
         git remote add github-upstream git@github.com:box/box-content-preview.git || return 1
     fi
 
+    # Fetch latest code with tags
+    git fetch github-upstream || return 1;
+
+    # Update to latest code on GitHub master
+    git checkout master || return 1
```
Comment (DanDeMicco) [inline build/upgrade_annotations.sh]: Comment here is incorrect. Shouldnt this be reset --hard upstream/master ?

Comment (DanDeMicco) [reply build/upgrade_annotations.sh]: Also side note you can look into the -e flag instead of the || return 1

File: build/upgrade_annotations.sh [hunk 2]
```diff
@@ -50,6 +50,10 @@ reset_to_master() {
         git remote add github-upstream git@github.com:box/box-content-preview.git || return 1
     fi
 
+    # Fetch latest code
+    git fetch github-upstream || return 1;
+    git checkout master || return 1
+
     # The master branch should not have any commits
     if [[ $(git log --oneline ...github-upstream/master) != "" ]] ; then
         echo "----------------------------------------------------"
```
Comment (DanDeMicco) [inline build/upgrade_annotations.sh]: Change this error message to say something like you have extra commits in your master branch

Comment (pramodsum) [reply build/upgrade_annotations.sh]: 👍

**Pull request merged**